### PR TITLE
Wkwebview

### PR DESCRIPTION
**Pull request by dylanvorster**
Adds WKWebview support for loading the app after updating

### CODE REVIEW

File: src/ios/AppUpdate.m [hunk 1]
```diff
@@ -115,18 +116,19 @@ - (void) doLoad {
         NSString *cordovaJs = [[NSBundle mainBundle] pathForResource:@"www/cordova" ofType:@"js"];
         NSString *fullUri = [NSString stringWithFormat:@"%@?cordova_js_bootstrap_resource=%@", uri, cordovaJs];
         NSURL *url = [NSURL URLWithString:fullUri relativeToURL:baseURL];
-
-        dispatch_async(self.serialQueue, ^{
+        
+        dispatch_async(dispatch_get_main_queue(), ^{
```
Comment (WesselWessels) [inline src/ios/AppUpdate.m]: For interest sake, is the main queue still needed?

Comment (dylanvorster) [reply src/ios/AppUpdate.m]: yes

File: src/ios/AppUpdate.m [hunk 2]
```diff
@@ -1,4 +1,6 @@
 #import "AppUpdate.h"
+#import <objc/message.h>
```
Comment (rkistner) [inline src/ios/AppUpdate.m]: I'm guessing this can be removed?

Comment (dylanvorster) [reply src/ios/AppUpdate.m]: probably

**Pull request merged**